app/src/main/res/layout/screen_onboarding.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<org.simple.clinic.onboarding.OnboardingScreen xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  tools:ignore="UnusedResources">
+
+  <FrameLayout
+    android:id="@+id/onboarding_logo_container"
+    style="@style/Clinic.LoginLogoContainer">
+
+    <TextView
+      android:id="@+id/onboarding_logo"
+      style="@style/Clinic.LoginLogo"
+      android:text="@string/app_name" />
+  </FrameLayout>
+
+  <android.support.v4.widget.NestedScrollView
+    android:id="@+id/onboarding_scrolling_container"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_alignParentTop="true"
+    android:fillViewport="true">
+
+    <LinearLayout
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:gravity="center_horizontal"
+      android:orientation="vertical"
+      android:paddingBottom="@dimen/spacing_96"
+      android:paddingEnd="@dimen/spacing_24"
+      android:paddingLeft="@dimen/spacing_24"
+      android:paddingRight="@dimen/spacing_24"
+      android:paddingStart="@dimen/spacing_24"
+      android:paddingTop="@dimen/spacing_128">
+
+      <!-- We are using "background" instead of "src" for the onboarding images because we want to get the shadow (on api 21+). We can do this because we set
+      width and height of the ImageView to match the aspect ratio of the images exactly. Otherwise, we have to workaround it by wrapping it in a CardView and
+      that seems unnecessary for now because the images are static and have a fixed aspect ratio
+      -->
```

Comment:
- Are you suggesting we set the width and height to `wrap_content`? I am fine with doing that, it just seemed a little overkill in this case since the layouts are completely static.
- Setting the images as `src` with a white background is not an option since they are PNGs and have rounded corners with transparent backgrounds. The corners of the onboarding images will not look good.